ingestion/src/metadata/data_insight/processor/web_analytic_report_data_processor.py

```diff
@@ -143,10 +159,19 @@ def _refine_entity_event(self) -> Generator[dict, WebAnalyticEventData, None]:
                         "`tags` attribute not supported for entity type",
                     )
 
+                try:
+                    entity_href = re.search(re_pattern, event.eventData.fullUrl).group(
+                        1
+                    )
+                except IndexError:
+                    entity_href = None
+
                 refined_data[split_url[1]] = {
-                    "entityType": entity_obj.entity_type.title(),
+                    "entityType": entity_type[0].upper()
+                    + entity_type[1:],  # Normalise entity type
```

Comment:
Can we use this?
```suggestion
                    "entityType": ENTITIES[entity_obj.entity_type],
```
I think it is a better approach instead of manually normalizing it or we can also move the normalization into a method that can be reused in the future.